devito/core/gpu_openmp.py

```diff
@@ -76,6 +77,10 @@ def _map_alloc(cls, f):
         return cls.lang['map-enter-alloc'](f.name, ''.join('[0:%s]' % i
                                                            for i in cls._map_data(f)))
 
+    @classmethod
+    def _map_present(cls, f):
+        raise NotImplementedError
```

Comment:
On slack, you gave a detailed explanation as to why you had to drop `present`. I think it would be useful to include that rational as a comment here and/or add an issue to GitHub as it is surely going to come up again as we continue to refine/optimize OpenACC support and you don't want to remember/re-explain to others (thinking about hackathons etc).

Reply:
good point, I'm adding it